src/components/Dashboard/DashboardItem/styles.js

```diff
@@ -0,0 +1,75 @@
+import { makeStyles } from "@material-ui/styles";
+
+export default makeStyles((theme) => ({
+    tripleDotMenu: {
+        marginLeft: "auto",
+    },
+    avatar: {
+        background: (props) => props.color,
+        height: theme.spacing(3),
+        width: theme.spacing(3),
+    },
+    cardHeaderDefault: {
+        background: theme.palette.white,
+    },
+    avatarIcon: {
+        color: (props) => theme.palette.white,
+        backgroundColor: (props) => props.color,
+        height: theme.spacing(2),
+        width: theme.spacing(2),
+    },
+    cardHeaderContent: {
+        width: "75%",
+    },
+    cardHeaderText: {
+        color: theme.palette.primary.main,
+    },
+    dashboardCard: {
+        display: "flex",
+        flexDirection: "column",
+        marginTop: theme.spacing(2),
+
+        width: (props) => props.width,
+        height: (props) => props.height,
```

Comment:
This is interesting! I probably missed it but how are props getting passed to here?